runtime/src/swam/runtime/internals/tracer/InFileTracer.scala

```diff
@@ -0,0 +1,65 @@
+package swam
+package runtime
+package internals
+package tracer
+
+import config._
+import java.io._
+import java.text.SimpleDateFormat
+import java.util._
+import java.util.UUID.randomUUID
+import scala.collection.mutable.ListBuffer
+/** Write events log file
+ * Every log file is grouped by the engine execution scope
+  */
+class InFileTracer(val conf: EngineConfiguration) extends Tracer{
```

Comment:
For this I think it would be better to use the `fs2-io` library to write the files.